src/libraries/Directory.Build.targets

```diff
@@ -274,6 +274,7 @@
 
   <PropertyGroup>
     <SkipLocalsInit Condition="'$(SkipLocalsInit)' == '' and '$(MSBuildProjectExtension)' == '.csproj' and '$(IsNETCoreAppSrc)' == 'true' and '$(TargetFramework)' == '$(NetCoreAppCurrent)'">true</SkipLocalsInit>
+    <VersionSuffix>$(_PreReleaseLabel)$(_BuildNumberLabels)</VersionSuffix>
```

Comment:
Usually we set the VersionSuffix in Versions.props. Why here only for libraries?

Reply:
VersionSuffix should be empty when stablizePackageVersion is set. We require this because of this library specific error check. https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.targets#L706
We have the same thing done in 3.1 as well. cc @ericstj 
Not sure why this error check is here, we should probably resolve this for 6.0

Reply:
So should we follow-up on this?

Reply:
https://github.com/dotnet/arcade/issues/6305